static/app/views/organizationStats/usageTable/index.tsx

```diff
@@ -157,6 +158,8 @@ export const CellStat = styled('div')`
   align-items: center;
   font-variant-numeric: tabular-nums;
   justify-content: right;
+  padding-top: 9px;
```

Comment:
what was the padding here previously?

Reply:
16px -> but with the added button width it looked weird